### PR TITLE
Update winit dependency to 0.19 and glutin to 0.20

### LINKED ISSUE #2743: Update Winit dependency
Short info header:
- GFX version: 0.1.0
- OS: Linux
- GPU: GTX 1060

I got a winit version mismatch while compiling my code (winit 0.19.1 vs winit 0.18.1). I fixed it by downgrading winit in my Cargo.toml, but would we get a version bump on the dependencies?

### PR DESCRIPTION
**Pull request by Abendstolz**
Fixes #2743 

PR checklist:
- [x] `make` succeeds (on *nix)
- [~] `make reftests` succeeds
- [x] tested examples with the following backends: vulkan
- [x] `rustfmt` run on changed code

Regarding reftests... I hope that this means it succeeded:
screenshot_23 04 2019-15 06 17

### CODE REVIEW

**Comment by kvark** [comment]
Thank you!
bors r+

File: src/warden/src/bin/reftest.rs [hunk 1]
```diff
@@ -239,7 +239,7 @@ fn main() {
         println!("Warding GL headless:");
         let events_loop = glutin::EventsLoop::new();
         let context =
-            glutin::Context::new(&events_loop, glutin::ContextBuilder::new(), false).unwrap();
+            glutin::Context::new_headless(&events_loop, glutin::ContextBuilder::new(), glutin::dpi::PhysicalSize::new(0.0, 0.0)).unwrap();
```
Comment (Abendstolz) [inline src/warden/src/bin/reftest.rs]: @kvark Not sure if this zero-sized context is appropriate

File: src/backend/gl/src/window/glutin.rs [hunk 2]
```diff
@@ -36,7 +36,7 @@
 //!
 //! fn main() {
 //!     let events_loop = EventsLoop::new();
-//!     let context = Context::new(&events_loop, ContextBuilder::new(), false)
+//!     let context = Context::new_headless(&events_loop, ContextBuilder::new(), glutin::dpi::PhysicalSize::new(0.0, 0.0))
```
Comment (Abendstolz) [inline src/backend/gl/src/window/glutin.rs]: @kvark Not sure if this zero-sized context is appropriate

**Comment by kvark** [comment]
bors r-
good point

File: src/warden/src/bin/reftest.rs (hunk 1 above)
Comment (kvark) [reply src/warden/src/bin/reftest.rs]: did you run reftests with it?

Comment (Abendstolz) [reply src/warden/src/bin/reftest.rs]: Yes, but see screenshot. I am not sure what to make of it.

Comment (kvark) [reply src/warden/src/bin/reftest.rs]: is this matching what you get without this PR?

Comment (Abendstolz) [reply src/warden/src/bin/reftest.rs]: yes

**Comment by Abendstolz** [comment]
@kvark As I can't see `bors` output, what can I do to get this merged?

**Comment by kvark** [comment]
Looks like Travis OSX is having a bad time - https://travis-ci.org/gfx-rs/gfx/jobs/523567981
bors retry

**Comment by kvark** [comment]
bors retry

**Comment by kvark** [comment]
bors retry

**Pull request merged**